CMakeLists.txt

```diff
@@ -54,13 +54,8 @@ if(BUILD_DFU)
   set(BUILD_DFU true)
 endif()
 
-option(WATCH_COLMI_P8 "Build for the Colmi P8" OFF)
-set(TARGET_DEVICE "PineTime")
-
-if(WATCH_COLMI_P8)
-  set(TARGET_DEVICE "Colmi P8")
-  add_definitions(-DWATCH_P8)
-endif()
+set(TARGET_DEVICE "PINETIME" CACHE STRING "Target device")
+set_property(CACHE TARGET_DEVICE PROPERTY STRINGS PINETIME P8A P8B)
```

Comment:
I don't know CMake that well, but I think this needs updating as it has the old P8A and P8B labels.

Reply:
Correct, fixed. Thank you.